src/main/java/org/quiltmc/installer/gui/swing/AbstractPanel.java

```diff
@@ -104,6 +105,55 @@ static void populateLoaderVersions(JComboBox<String> comboBox, List<String> load
 		comboBox.setEnabled(true);
 	}
 
+	protected @Nullable List<JComponent> createBeaconOptOut() {
+		if (Boolean.parseBoolean(System.getenv().get("QUILT_DISABLE_BEACON")) ||
+			Boolean.parseBoolean(System.getenv().get("CI")) ||
```

Comment:
I don't think we should bother with this check: we aren't recommending that users use this variable, and we're not going to be running the installer in ci

Reply:
I don't really see this check causing problems. My intention was to match the loader's checks as closely as possible. Honestly, there will be someone out there who set this to disable telemetry. Or maybe it's there because of some other application that needs it. Either way, this checkbox won't do much in this situation and having it there would be confusing.